### PR TITLE
adding font

### PR DESCRIPTION
**Pull request by mstone6769**

### CODE REVIEW

File: lib/src/general-styles/_lists.scss [hunk 1]
```diff
@@ -122,11 +122,13 @@ $large-list: 200px;
 $xlarge-list: 270px;
 
 @mixin dl-type($size: $short-list) {
-  > dt {
-    width: $size;
-  }
-  > dd {
-    margin-left: $size;
+  @media #{$mq-sm-screen} {
```
Comment (mcastre) [inline lib/src/general-styles/_lists.scss]: good

File: lib/src/iui-general.scss [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 @import "_iui-variables";
 
+// Bootstrap Variables
+
+$icon-font-path: '/$iui-general/fonts/bootstrap/';
```
Comment (mcastre) [inline lib/src/iui-general.scss]: so we decided to keep the icon font library?

Comment (mstone6769) [inline lib/src/iui-general.scss]: @mcastre it's in bootstrap - so without it, it's possible the app will get 404s

Comment (mcastre) [inline lib/src/iui-general.scss]: in Predict our path is themes/fonts/bootstrap. Will that be an issue?

Comment (mstone6769) [inline lib/src/iui-general.scss]: this should replace the need for those files

Comment (mcastre) [inline lib/src/iui-general.scss]: I hope so, because I dont want to bring in 2 sets of fonts

Comment (mcastre) [inline lib/src/iui-general.scss]: have you tested this in Empower?

**Pull request merged**